View/main_window.py

```diff
@@ -1,6 +1,6 @@
 from datetime import datetime
 
-from PySide6.QtCore import Qt, QCoreApplication, QSettings
+from PySide6.QtCore import Qt, QCoreApplication, QSettings, Signal, QObject
```

Comment:
The import QObject is unused, and can be reviewed. While you're here, the datetime import can also be removed as it's not used anymore.